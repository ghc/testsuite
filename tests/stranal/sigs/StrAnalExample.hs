-- Just an example on how to create tests that test the strictness analizer

module StrAnalExample where

foo x = x
