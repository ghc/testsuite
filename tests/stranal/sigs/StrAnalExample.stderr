
==================== Strictness signatures ====================
StrAnalExample.foo: <S,1*U>


